l1-contracts/contracts/bridgehub/Bridgehub.sol

```diff
@@ -662,7 +662,23 @@ contract Bridgehub is IBridgehub, ReentrancyGuard, Ownable2StepUpgradeable, Paus
         bytes32 _assetId,
         address _depositSender,
         bytes calldata _data
-    ) external payable override onlyAssetRouter onlyL1 {}
+    ) external payable override onlyAssetRouter onlyL1 {
+        delete settlementLayer[_chainId];
+
+        IStateTransitionManager(stateTransitionManager[_chainId]).bridgeClaimFailedBurn({
+            _chainId: _chainId,
+            _assetInfo: _assetId,
+            _prevMsgSender: _depositSender,
```

Comment:
the name of `_prevMsgSender` is somewhat incorrect here. `_depositSender` is a more correct name, i.e. this method is permissionless and can be called by anyone, it is just that the address of the `_depositSender` will be passed